requre/polarion/__init__.py

```diff
@@ -27,7 +26,14 @@
 sessions.Session.send = RequestResponseHandling.decorator(
     item_list=[1])(sessions.Session.send)
 
-tmt.utils.check_git_url = Guess.decorator_plain()(tmt.utils.check_git_url)
+# The function moved to 'tmt.utils.git'
+# https://github.com/teemtee/tmt/pull/2983
+try:
+    import tmt.utils
+    tmt.utils.check_git_url = Guess.decorator_plain()(tmt.utils.check_git_url)
+except AttributeError:
+    import tmt.utils.git
```

Comment:
Looks like a redundant import to me. imho it should either be `from tmt.utils import ..` and `from tmt.utils.git import ..` or just one `import tmt.utils`, which can be left on top as it was.